noxfile.py

```diff
@@ -106,11 +93,70 @@ def requirements(session):
             filename,
             "--resolution",
             resolution,
-            *flags,
         )
 
 
-sphinx_commands = (
+pytest_command: tuple[str, ...] = (
+    "pytest",
+    "--pyargs",
+    "--durations=5",
+    "--tb=short",
+    "-n=auto",
+    "--dist=loadfile",
+)
+
+with_doctests: tuple[str, ...] = ("--doctest-modules", "--doctest-continue-on-failure")
+
+with_coverage: tuple[str, ...] = (
+    "--cov=plasmapy",
+    "--cov-config=pyproject.toml",
+    "--cov-append",
+    "--cov-report xml:coverage.xml",
+)
+
+skipslow: tuple[str, ...] = ("-m", "not slow")
+
+test_specifiers: list[nox._parametrize.Param] = [
+    nox.param("all", id="all"),
+    nox.param("with code coverage", id="cov"),
+    nox.param("skip slow tests", id="skipslow"),
+    nox.param("lowest-direct", id="lowest-direct"),
+]
+
+
+@nox.session(python=supported_python_versions)
+@nox.parametrize("test_specifier", test_specifiers)
```

Comment:
One neat thing about nox is that we can parametrize test sessions with an API that is very close to `@pytest.mark.parametrize`.

The `id` fields here make it so that `test_specifier=` doesn't show up in the list of parametrized nox sessions.